server/app/models/host_node.rb

```diff
@@ -28,6 +28,7 @@ class HostNode
   field :last_seen_at, type: Time
   field :agent_version, type: String
   field :docker_version, type: String
+  field :connected_at, type: DateTime
```

Comment:
The `:connected_at.lt => Time.parse(...)` comparisons do not seem to work at all if this is `type: Time`.